inspect.yaml

```diff
@@ -51,6 +51,42 @@ paths:
               schema:
                 $ref: "#/components/schemas/Error"
 
+  /:
+    post:
+      operationId: inspect_post
+      summary: Inspect DApp state via POST
+      description: |
+        This method sends an inspect-state request to the DApp backend passing the payload string available in the request body.
+        The payload string should be URL-encoded; the inspect server will decode the string to UTF-8.
+        If the DApp frontend needs to pass a binary string to the backend then it is advised to use the base64 encoding.
```

Comment:
This can be removed as well. In the GET version of this function, we should change this sentence to: "If the DApp frontend needs to pass a binary string to the backend then it is advised to use the POST method".

Reply:
Done at [82a6bc8](https://github.com/cartesi/openapi-interfaces/pull/2/commits/82a6bc8fac6c1ccd08c91e43e382d630a1dc5627).